src/xdocs/config_header.xml

```diff
@@ -350,6 +352,36 @@ line 6: ^\W*$
         </source>
 
         <p>
+          For regex containing <code>"\n\n"</code>
+        </p>
+        <source>
+&lt;module name=&quot;RegexpHeader&quot;&gt;
+  &lt;property
+    name=&quot;header&quot;
+    value=&quot;^package .*\n\n.*&quot;/&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          <code>"\n\n"</code> will be treated as '^$' and will forcefully
+          expect the line to be empty. For example -
+        </p>
+        <source>
+package com.some.package;
+/**
+ *
+*/
+public class ThisWillFail { }
+        </source>
+        <p>
+          would fail for the regex above. Expected -
+        </p>
+        <source>
+package com.some.package;
+
+public class ThisWillPass { }
```

Comment:
In your before example you had a javadoc. In this after example, there is no javadoc.
It is slightly misleading in that it makes it sound like you are forcing users to remove javadoc.

Either restore javadoc in after example, or remove it in before example so that `package` and `public` are on consecutive lines.